demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -326,3 +333,30 @@ def parse_error_line(self, err) -> str:
             return (
                 'Failed: {} failed.\nMissing {} in {}'.format(self.file_path, str(key_from_error), "root",
                                                               ))
+
+    @staticmethod
+    def get_content_parser(file_path: str) -> Callable[[str], str]:
```

Comment:
feels to me this should be in tools, what do you think?
also the func name is not very indicative, what about something like `get_content_file_type_dump` ?